packages/arb-validator-core/mockbridge/messagesChallenge.go

```diff
@@ -123,6 +123,15 @@ func (c *MessagesChallenge) OneStepProofTransactionMessage(
 	return nil
 }
 
+func (c *MessagesChallenge) OneStepProofTransactionBatchMessage(
+	ctx context.Context,
+	lowerHashA common.Hash,
+	lowerHashB common.Hash,
+	msg message.DeliveredTransactionBatch,
+) error {
+	return nil
```

Comment:
This looks suspiciously like a not-yet-implemented function.  If it's supposed to always return no error, leave a comment explaining why.

Reply:
The mockbridge stuff never was merged so all of the stuff in that package is unimplemented.